release.md

```diff
@@ -1,17 +1,11 @@
 ---
 title: Releases
-layout: default
+layout: page
 pagination:
   data: collections.release
   size: 50
   alias: posts
 ---
-<!-- Header -->
-<header class="masthead d-flex p-5">
-  <div class="container text-center my-auto">
-    <img style="width: 50%; margin-bottom: 3rem;" src="{{ '/assets/svg/web_Logo_Header_Color_300x110px.svg' | url }}">
-    <p class="lead mb-0"><strong>WPE WebKit</strong> allows <strong>embedders</strong> to create simple and performant systems based on <strong>Web</strong> platform technologies. It is designed with hardware acceleration in mind, leveraging common 3D graphics APIs for best <strong>performance</strong>.</p>
```

Comment:
From what I see, the header is not the same here. But I guess that is the intended purpose.

Reply:
This was a leftover from the old design of the website.